README.md

```diff
@@ -1,2 +1,114 @@
 # alsf-scpca
 Management and analysis tools for ALSF Single-cell Pediatric Cancer Atlas data.
+
+- [Environment Setup](#environment-setup)
+  - [Nextflow](#nextflow)
+    - [Installing Nextflow](#installing-nextflow)
+    - [Nextflow Tower](#nextflow-tower)
+  - [Docker](#docker)
+  - [AWS](#aws)
+    - [Installing AWS Command line tools](#installing-aws-command-line-tools)
+    - [Configuring your AWS credentials](#configuring-your-aws-credentials)
+    - [AWS infrastructure](#aws-infrastructure)
+- [Running Workflows](#running-workflows)
+
+## Environment Setup
+
+### Nextflow
+
+Most of the workflows in this repository are run via [Nextflow](https://www.nextflow.io).
+
+#### Installing Nextflow
+
+You can install Nextflow by following the [instructions on their page](https://www.nextflow.io/docs/latest/getstarted.html#installation).
+Be sure to follow the second step of their instructions: move the `nextflow` file to a directory in your `$PATH` for easy access.
+
+Alternatively, you can install `nextflow` via `conda` or `brew` (it may also be available in other package managers):
+- `conda install -c bioconda nextflow`
+- `brew tap brewsci/bio; brew install nextflow`
+
+#### Nextflow Tower
+
+Optionally, you can track your nextflow workflows in progress with [Nextflow Tower](https://tower.nf/).
+To use this, you will need to create an account on https://tower.nf/ (using your github account is likely simplest), then get your token value from https://tower.nf/tokens.
+You will then want to create an environment variable with this token value:
+`export TOWER_ACCESS_TOKEN=<MYTOWERTOKEN>`
+This line can be entered at the command line, but you will likely want to add it to your shell initialization file (`~/.bash_profile`, `~/.zshrc`, etc.)
+
+### Docker
+
+All of the workflows are designed to run via Docker containers for each tool. It is therefore critical that you have Docker installed for any local execution. For downloads and installation, go to https://www.docker.com/products/docker-desktop
```

Comment:
But the Docker does not have Nextflow or AWS?

Reply:
Hmm, I think one of them does, but that is irrelevant here. Most of the docker tools are single-use, as in they are designed to run just one tool. Nextflow runs a whole bunch of different ones, but that is mostly automatic. You don't need to (and should not) build or pull any docker images manually to run anything.

Reply:
Gotcha. Okay. So if I install Nextflow and AWS I should be good to go.

Reply:
In this case, can you move this toward the end of the document or as a subnote underneath Nextflow?  This way I don't accidentally think I'll have to install this too (even though I realize you have no "Install Docker" section).

Reply:
You *do* have to install Docker. "It is therefore critical that you have Docker installed for any local execution." But you just have to have the engine installed, and that is best covered by the linked docs.

Reply:
OMG. Right.  Never mind.